saleor/graphql/core/types/filter_input.py

```diff
@@ -183,6 +188,21 @@ class Meta:
         )
 
 
+class FloatFilterInput(graphene.InputObjectType):
+    eq = graphene.Float(description=FilterInputDescriptions.EQ, required=False)
+    one_of = NonNullList(
+        graphene.Float, description=FilterInputDescriptions.ONE_OF, required=False
+    )
+    range = PriceRangeInput(description=FilterInputDescriptions.RANGE, required=False)
+
+    class Meta:
+        description = (
+            "Define the filtering options for integer fields."
+            + ADDED_IN_311
```

Comment:
314?

Reply:
Description also sounds wrong: `"Define the filtering options for integer fields."` (integer)